Dockerfile

```
@@ -207,6 +229,86 @@ RUN mkdir src &&\
     cmake \
         --build src/build \
         --target install \
+#	-j$(nproc) \
     &&\
     ln -s ${__prefix}/bin/thisdd4hep.sh ${__ldmx_env_script_d__}/thisdd4hep.sh &&\
     rm -r src
+
+###############################################################################
+# LHAPDF
+#
+# Needed for GENIE
+#
+###############################################################################
+LABEL lhapdf.version="6.5.2"
+ENV PYTHON_VERSION=3
+RUN mkdir src &&\
+    ${__wget} https://lhapdf.hepforge.org/downloads/?f=LHAPDF-6.5.2.tar.gz |\
+     ${__untar} &&\
+    cd src &&\
+    ./configure --prefix=${__prefix} &&\
+    make -j$(nproc) &&\
+    make -j$(nproc) install &&\
```

If LHAPDF isn't horrifically designed, you should be able to omit the first line and `make` will know to build before trying to install.

```suggestion
    make -j install &&\
```

---

Well, one can hope ... but yes, removed the dual make.

---

This one wasn't needed. The one for GENIE was. <justified fear>